pkgs/cortesi/modd/registry.yaml

```diff
@@ -0,0 +1,18 @@
+packages:
+  - type: github_release
+    repo_owner: cortesi
+    repo_name: modd
+    description: "A flexible developer tool that runs processes and responds to filesystem changes"
+    asset: "modd-{{.Version|trimV}}-{{.OS}}{{.Arch}}.{{.Format}}"
+    format: tgz
+    format_overrides:
+      - goos: windows
+        format: zip
+    replacements:
+      darwin: osx
+      amd64: 64
+    files:
+      - name: modd
+        src: 'modd-{{.Version|trimV}}-{{.OS}}{{.Arch}}/modd'
+    rosetta2: true
+    supported_if: not ((GOOS == "linux" or GOOS == "windows") and GOARCH == "arm64")
```

Comment:
I wasn't sure if there was a preferred way to say that there is no binary for linux/arm64 or windows/arm64.